### PR TITLE
duchess at scala.io

### PR DESCRIPTION
**Pull request by P4uline**
# Bienvenue sur les contributions Duchess ! 🦄

## Contenu de la Pull Request

- Description: 

- Lien vers la github issue si il y a:

[Lien vers les consignes de contribution](https://github.com/DuchessFrance/DuchessFrance.github.io/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: docs/_posts/2024/2024-03-31-scala-io-nantes.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+Going in conferences, meetups.
+Pair programming, mob programming sessions with your team.
+
+🇬🇧 ***My first year in Scala*** <br/>
+[![My first year in Scala](https://img.youtube.com/vi/Jyn2l1nhwZE/0.jpg){: width="600"}](https://www.youtube.com/watch?v=Jyn2l1nhwZE)
+
+# Focus on Johanna's and Mehdi's talk
+
+Starting with a few dance steps, Johanna and Mehdi presented [Kapoeira](https://github.com/lectra-tech/kapoeira/). 
+Based on cucmber-scala, Kapoeira is an open source library which implements acceptance tests on kafka streams.
```
Comment (gansheer) [inline docs/_posts/2024/2024-03-31-scala-io-nantes.md:39]:
```suggestion
Based on cucumber-scala, Kapoeira is an open source library which implements acceptance tests on kafka streams. 
```

File: docs/_posts/2024/2024-03-31-scala-io-nantes.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+val examintionReport = examine(medor)(using sumon[Clinic[Animal]])
+```
+
+🇬🇧 ***Contravariance: intuition building from first principles*** <br/>
+[![Contravariance: intuition building from first principles](https://img.youtube.com/vi/A7t3b0kymFM/0.jpg){: width="600"}](https://www.youtube.com/watch?v=A7t3b0kymFM)
+
+# Focus on Olya's talk
+
+Olya shared her experience on Ukraine's Scala community building and about the book club organized on the basis of Intelias, where I currently work.
```
Comment (gansheer) [inline docs/_posts/2024/2024-03-31-scala-io-nantes.md:96]: J'ai essayé de trouver ce qu'étais "Intelias" sans succès. J'ai l'impression que soit il y a un soucis de contexte soit c'est l'orthographe :smile:

File: docs/_posts/2024/2024-03-31-scala-io-nantes.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+date: "2024-04-02"
+categories:
+  - "code"
+  - "event"
+---
+
+| ![Duchess in Scala.io](/assets/sponsors/scalaio.webp){: width="300"} | Duchess was at Scala.io for the 2024 edition in Nantes |
+
+
+A couple of weeks ago was the scala conference in Nantes. This article focus of the scala women speakers present at the event.
```
Comment (sludovicdelys) [inline docs/_posts/2024/2024-03-31-scala-io-nantes.md:15]: Proposition de formulation : "A couple of weeks ago, a significant number of female speakers participated at the Scala.IO conference held in the city of Nantes."

File: docs/_posts/2024/2024-03-31-scala-io-nantes.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+Also Scala.io organizers offered 3 tickets to the Duchess France community.
+
+# Focus on Monica's talk
+
+Great feedback from Monica. She shared with us about her first year of java and gave very good hints.
```
Comment (sludovicdelys) [inline docs/_posts/2024/2024-03-31-scala-io-nantes.md:20]: Proposition de formulation : "Monica shared with us valuable knowledge that she acquired while learning Scala for the first time".

File: docs/_posts/2024/2024-03-31-scala-io-nantes.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+**Help others to help you**<br/>
+Junior people should focus on providing context, goal, and more generally be great communicators.
+
+**Let us fail**<br/>
+Let junior developers making and owning mistakes.
+Senior people should let junior people (or new comers in the teams) fail so they can take ownership of their mistakes and learn from them. If you're a senior developer don't solve the problem for you're team mates but rather teach the steps to go through to tackle the problem. Enable the team to solve problems, even it takes a little long to solve these problems.
```
Comment (sludovicdelys) [inline docs/_posts/2024/2024-03-31-scala-io-nantes.md:27]: Correction : Senior people should let junior people (or newcomers in the teams) fail so they can take ownership of their mistakes and learn from them. If you're a senior developer don't solve the problem for your teammates but rather teach them the steps to tackle the problem. Enable the team to solve their own problems, even it takes a little longer.

File: docs/_posts/2024/2024-03-31-scala-io-nantes.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+
+**Find a community**<br/>
+Going in conferences, meetups.
```
Comment (sludovicdelys) [inline docs/_posts/2024/2024-03-31-scala-io-nantes.md:30]: Going `to` conferences `and` meetups

File: docs/_posts/2024/2024-03-31-scala-io-nantes.md (hunk 2 above)
Comment (sludovicdelys) [inline docs/_posts/2024/2024-03-31-scala-io-nantes.md:96]: Proposition de formulation : Olya shares her exploration of the Ukrainian Scala community and reflects on the lessons learned in the process of community building while setting up a book club within her company Intellias.

File: docs/_posts/2024/2024-03-31-scala-io-nantes.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+```
+
+🇬🇧 ***Contravariance: intuition building from first principles*** <br/>
+[![Contravariance: intuition building from first principles](https://img.youtube.com/vi/A7t3b0kymFM/0.jpg){: width="600"}](https://www.youtube.com/watch?v=A7t3b0kymFM)
+
+# Focus on Olya's talk
+
+Olya shared her experience on Ukraine's Scala community building and about the book club organized on the basis of Intelias, where I currently work. 
+Given the war with Russia, it is now physically unsafe for people to gather, so they have created an online community and meet each week to discuss one chapter of the book. 
+This format allows people who were forced to leave the country, as well as invited guests, to join. In addition, they experimented with different formats and added "Scala Breaks" - small reports with examples of interesting problems or simply a scala of related topics.
```
Comment (sludovicdelys) [inline docs/_posts/2024/2024-03-31-scala-io-nantes.md:98]: 💡 Proposition de formulation : This format allows people who were forced to leave the country, as well as guests to join the meetings.

File: docs/_posts/2024/2024-03-31-scala-io-nantes.md [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+
+🇬🇧 ***Contravariance: intuition building from first principles*** <br/>
+[![Contravariance: intuition building from first principles](https://img.youtube.com/vi/A7t3b0kymFM/0.jpg){: width="600"}](https://www.youtube.com/watch?v=A7t3b0kymFM)
+
+# Focus on Olya's talk
+
+Olya shared her experience on Ukraine's Scala community building and about the book club organized on the basis of Intelias, where I currently work. 
+Given the war with Russia, it is now physically unsafe for people to gather, so they have created an online community and meet each week to discuss one chapter of the book. 
+This format allows people who were forced to leave the country, as well as invited guests, to join. In addition, they experimented with different formats and added "Scala Breaks" - small reports with examples of interesting problems or simply a scala of related topics. 
+Thus, they have already read 5 books and more than 45 meetings.
```
Comment (sludovicdelys) [inline docs/_posts/2024/2024-03-31-scala-io-nantes.md:99]: Thus, they have already read 5 books and held more than 45 meetings.

**Review by sludovicdelys — COMMENTED** [review]
Coucou ! Je me suis permise de faire quelques propositions de formulation avec quelques petites corrections. Prends ce qui est pertinent pour l'article et j'espère que j'ai été un peu utile 😊

File: docs/_posts/2024/2024-03-31-scala-io-nantes.md (hunk 4 above)
Comment (monmcguigan) [inline docs/_posts/2024/2024-03-31-scala-io-nantes.md:20]: It is on Scala, not Java!

Comment (P4uline) [reply docs/_posts/2024/2024-03-31-scala-io-nantes.md]: Oops, fixed

**Pull request merged**